snippets/dart_2_snippets.md

```diff
@@ -0,0 +1,858 @@
+## Available Dart 2 Builder-Compatible Code Snippets
+
+Below is a list of all available Dart 2 builder-compatible over_react snippets and their<br>
+triggers for Webstorm/IntelliJ and VS Code. 
+
+| Trigger  | Content |
+| -------: | ------- |
+| `or_stateless`   | dart 2 stateless component skeleton |
+| `or_stateless_commented`   | dart 2 commented stateless component skeleton |
+| `or_abstract_stateless`  | dart 2 abstract stateless component skeleton |
+| `or_abstract_stateless_commented`  | dart 2 commented abstract stateless component skeleton |
+| `or_stateful`   | dart 2 stateful component skeleton |
+| `or_stateful_commented`   | dart 2 commneted stateful component skeleton |
+| `or_abstract_stateful`  | dart 2 abstract stateful component skeleton |
+| `or_abstract_stateful_commented`  | dart 2 commented abstract stateful component skeleton |
+| `or_props_mixin`  | dart 2 prop mixin skeleton |
+| `or_state_mixin`  | dart 2 state mixin skeleton |
+
+## WebStorm and IntelliJ Snippets
+
+### Installation Instructions
+
+1. Click "WebStrom" or "IntelliJ" and select "Preferences" 
+2. Click on the Editor drop down and select Live Templates
+3. Click the "+" button in the upper right corner and select "Template Group..."
+    1. Provide a meaningful name for your template group like "OverReactDart2"
+    2. Alternatively, you can select an existing group, click on the "+" button and<br>
+       select "Live Template" to add a new template to that group
+4. Click the "+" again once you've created or selected the template group to which you'd<br> 
+   like to add a new template and select "Live Template"
```

Comment:
#nit these first steps could be made a little more concise:
1. Open preferences and select "Editor" > "Live Templates"
1. (Optional) Create a new "OverReact Dart 2" group by clicking the "+" button in the upper right corner and selecting "Template Group..."
1. Select your new (or an existing) template group, click the "+" again, and select "Live Template..."